spec/models/manageiq/providers/infra_manager_spec.rb

```diff
@@ -25,4 +25,15 @@
       expect(described_class.ems_timeouts(:ems_redhat, :InVentory)).to eq [5.hours, nil]
     end
   end
+
+  describe '.clusterless_hosts' do
+    it "hosts with no ems" do
+      ems = FactoryBot.create(:ems_infra)
+      cluster = FactoryBot.create(:ems_cluster)
+      hosts = FactoryBot.create_list(:host, 2, :ext_management_system => ems)
+      FactoryBot.create_list(:host, 2, :ext_management_system => ems, :ems_cluster => cluster)
+
+      expect(ems.clusterless_hosts).to match_array(hosts)
```

Comment:
How about 1 host with a cluster and 1 clusterless host and assert only the latter is returned?

Reply:
Keenan pointed me at line 34 where it's creating a clustered host so we can be sure it's only returning the non-clustered hosts and not the clustered ones.  👍